dev-cpp/tbb/patches/tbb-2019.patchset

```diff
@@ -520,5 +520,99 @@ index dcf856a..baa9abd 100644
  #elif __linux__  // Order of these elif's matters!
  #define EXT __TBB_STRING(.so.TBB_COMPATIBLE_INTERFACE_VERSION)
 -- 
-2.15.0
+2.19.0
+
+
+
+
+From 8f22979d461865316e05515fcbb56b54e8a74271 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sat, 1 Sep 2018 22:49:59 +0200
+Subject: Haiku runtime_loader cant use textfiles, use symlinks instead
```

Comment:
The tests run fine without that patch. Got a compiler segmentation fault however:

```
g++ -o test_eh_algorithms.o -c -MMD -DDO_ITT_NOTIFY -g -O2 -DUSE_PTHREAD -m64 -mrtm   -DTEST_USES_TBB=1 -Wall -Wshadow -Wcast-qual -Woverloaded-virtual -Wnon-virtual-dtor -Wextra    -I../../src -I../../src/rml/include -I../../include -I. -include ../../src/test/harness_preload.h ../../src/test/test_eh_algorithms.cpp
../../src/test/test_eh_algorithms.cpp: In function 'void TestParallelLoopAux() [with ParForBody = SimpleParForBody; Partitioner = const tbb::simple_partitioner]':
../../src/test/test_eh_algorithms.cpp:162:6: internal compiler error: Segmentation violation
 void TestParallelLoopAux() {
      ^~~~~~~~~~~~~~~~~~~
libbacktrace could not find executable to open
Please submit a full bug report,
with preprocessed source if appropriate.
See <https://gcc.gnu.org/bugs/> for instructions.
../../build/common_rules.inc:86: recipe for target 'test_eh_algorithms.o' failed
make[1]: *** [test_eh_algorithms.o] Error 1
make[1]: Leaving directory '/sources/tbb-2018_U5/build/haiku_intel64_gcc_cc7.3.0_libc_kernel1_release'
Makefile:44: recipe for target 'test' failed
```